docs/baseline-protocol/mainnet.md

```diff
@@ -7,35 +7,27 @@ description: >-
 
 # Driving Mainnet Evolution
 
-The Mainnet is a \[ insert copy from the upcoming Medium article \]. 
+The Mainnet is a \[ insert copy from the upcoming Medium article \].
 
 ## Point to Point Messaging
 
-The Radish34 demo uses Whisper, which comes with standard Geth nodes, to 
+The Radish34 demo uses Whisper, which comes with standard Geth nodes, to
 
 ## Zokrates Support or Replacement
 
-While the baseline approach isn't solely about zero knowledge techniques, it uses them to ensure that all parties store the same data in their respective systems of record, execute business rules consistently, and enforce the correctness of Workflow Steps. 
+While the baseline approach isn't solely about zero knowledge techniques, it uses them to ensure that all parties store the same data in their respective systems of record, execute business rules consistently, and enforce the correctness of Workflow Steps.
 
-One aspect of this is Zokrates, a domain specific language \(DSL\) for taking logical functions and turning them into the "addition, subtraction, multiplication" math required by zero knowledge proofs. Zokrates is an open source project maintained by \[  \]. From the perspective of the enterprise, it has the disadvantage of being distributed under a \[  \] license, not Apache2 or MIT. 
+One aspect of this is Zokrates, a domain specific language \(DSL\) for taking logical functions and turning them into the "addition, subtraction, multiplication" math required by zero knowledge proofs. Zokrates is an open source project maintained by \[ \]. From the perspective of the enterprise, it has the disadvantage of being distributed under a \[ \] license, not Apache2 or MIT.
 
 For enterprises to adopt a DSL like Zokrates as part of their core IT stack, it needs to be GA'd, the problem of the license must be dealt with -- there are a number of ways to handle this -- and the project must be well supported at a level that can give enterprises confidence.
 
 ## Eth2
 
-The Baseline Protocol can provide utility for a variety of uses across industries today on the current \[  \] Ethereum. It avoids several of the key issues enterprises have with using "public" Ethereum. 
+The Baseline Protocol can provide utility for a variety of uses across industries today on the current \[ \] Ethereum. It avoids several of the key issues enterprises have with using "public" Ethereum.
```

Comment:
It would be good to mention any specific points around Eth2. Eth2 is aiming at several targets - privacy, scalability and improved consensus. Baseline protocol can be designed or furthered to adopt Eth2 standards and implications - some content along these lines would justify the title of this segment